src/main/java/zd/zero/waifu/motivator/plugin/integrations/ErrorReporter.kt

```diff
@@ -47,7 +47,7 @@ class ErrorReporter : ErrorReportSubmitter() {
         events: Array<out IdeaLoggingEvent>,
         additionalInfo: String?,
         parentComponent: Component,
-        consumer: Consumer<SubmittedReportInfo>
+        consumer: Consumer<in SubmittedReportInfo>
```

Comment:
I tested to make sure that this is backwards compatible with the older supported builds.

Reply:
Ohhh.... this is how you do it in Kt @@
#til